tools/generator/test/Fixtures.swift

```diff
@@ -1962,9 +1962,6 @@ sed \
         let (files, _) = Fixtures.files(in: pbxProj, parentGroup: mainGroup)
         let products = Fixtures.products(in: pbxProj, parentGroup: mainGroup)
 
-        let internalDirectoryName = "rules_xcp"
-        let workspaceOutputPath: Path = "Project.xcodeproj"
-
```

Comment:
I noticed the warnings for these unused variables. So, I went ahead and removed them.